src/Tribe/Main.php

```diff
@@ -527,6 +528,9 @@ public function load_text_domain() {
 		public function bind_implementations(  ) {
 			tribe_singleton( 'tec.main', $this );
 
+			// i18n
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: dac5a3da614956a99f2f262e8a4587ec</sub>